SAR_Training/English/Master/ISCE3_Sentinel1_RTC.ipynb

```diff
@@ -46,18 +46,20 @@
    "source": [
```

Comment:
<p>I think we can get rid of this corner reflector stuff in the generalized ISCE3 processing notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ASFOpenSARlab/opensarlab-notebooks/pull/206/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>